src/librustc/mir/interpret/error.rs

```diff
@@ -389,6 +389,10 @@ pub enum UnsupportedOpInfo<'tcx> {
     /// Free-form case. Only for errors that are never caught!
     Unsupported(String),
 
+    /// FIXME(#64506) Error used to work around accessing projections of
+    /// uninhabited types.
+    UninhabitedValue,
```

Comment:
Why is "unsupported" the right category for this?

Reply:
To me, it seemed similar to some of the other "unsupported" cases like `ReadForeignStatic`, `NoMirFor`, `DerefFunctionPointer`, `OutOfTls` etc in that it represents allowed behavior in a valid program that simply isn't implemented (or can't be implemented) in miri. I dismissed `InvalidProgramInfo` because this doesn't indicate an invalid program, however reading the doc comment, it sounds like that may actually be a more appropriate classification. Thoughts?

Reply:
Looks like @oli-obk is already working on removing this again so whatever.